src/main/scala-sbt-1.0/sbtprotoc/Compat.scala

```diff
@@ -14,9 +12,7 @@ private[sbtprotoc] trait Compat extends CacheImplicits { self: ProtocPlugin.type
   }
 
   protected object CacheArguments {
-    implicit val instance: JsonFormat[Arguments] = project(
-      (a: Arguments) => (a.includePaths, a.protocOptions, a.pythonExe, a.deleteTargetDirectory, a.targets),
-      (in: (Seq[File], Seq[String], String, Boolean, Seq[(File, Seq[String])])) => Arguments(in._1, in._2, in._3, in._4, in._5)
-    )
+    implicit val instance: JsonFormat[Arguments] =
+      caseClassArray(Arguments.apply _, Arguments.unapply _)
```

Comment:
The effect of that problem is json deserializing fails (with an internal exception) and the bug depends on the use of Unbuilder's "nextField", which the case formats don't use: https://github.com/eed3si9n/sjson-new/blob/master/core/src/main/boilerplate/sjsonnew/CaseClassFormats.scala.template. So we should be good on this front.

Btw it's being addressed in https://github.com/eed3si9n/sjson-new/pull/72, which is pending.